### PR TITLE
Add prettier & lint to blui-ci job

### PR DESCRIPTION
**Pull request by jeffvg**
Fixes BLUI-5832.

#### Changes proposed in this Pull Request:

- add prettier and lint checks to job
- open PR once this is merged. https://github.com/etn-ccis/blui-react-native/pull/93
-

#### Screenshots / Screen Recording (if applicable)

-

#### To Test:

-

#### Any specific feedback you are looking for?

-

### CODE REVIEW

File: .github/workflows/blui-ci.yml [hunk 1]
```diff
@@ -32,10 +32,8 @@ jobs:
                   node-version: ${{ matrix.node-version }}
                   cache: 'yarn'
             - run: yarn
-            # - run: yarn docs-prettier
-            #   working-directory: docs
-        #   - run: yarn docs-lint
-        #     working-directory: docs
+            - run: yarn prettier:check
+            - run: yarn docs-lint
```
Comment (ArshdeepSingh3-eaton) [inline .github/workflows/blui-ci.yml:36]: shouldn't this be `yarn lint`

Comment (jeffvg) [reply .github/workflows/blui-ci.yml:36]: `docs-lint` and `lint` are the same thing. I can change it

Comment (ArshdeepSingh3-eaton) [reply .github/workflows/blui-ci.yml]: `docs-lint `runs for only docs project 
`lint` runs for all projects

**Review by ArshdeepSingh3-eaton — APPROVED** [review]

**Review by surajeaton — APPROVED** [review]
Looks good.

**Pull request merged**